common/views/components/VenueHours/VenueHours.js

```diff
@@ -96,6 +91,28 @@ const VenueHours = ({ venue, isInList }: Props) => {
     },
   };
 
+  const jauntyRef = useRef(null);
+
+  function setRandomClipPath() {
+    const topLeft = randomPx();
+    const topRight = randomPx();
+    const bottomRight = randomPx();
+    const bottomLeft = randomPx();
+
+    ['clipPath', 'webkitClipPath'].forEach(property => {
+      jauntyRef.current.style[property] = `polygon(
+        ${topLeft} ${topLeft},
+        calc(100% - ${topRight}) ${topRight},
+        100% calc(100% - ${bottomRight}),
+        ${bottomLeft} 100%
+      )`;
+    });
+  }
+
+  useEffect(() => {
+    setRandomClipPath();
```

Comment:
out of interest - why does this need to happen on the client?

Reply:
See [here](https://github.com/wellcometrust/wellcomecollection.org/pull/4383#issue-270033769)